packages/core-components/src/lib/unsorted/ui/Tabs/Tabs.svelte

```diff
@@ -16,23 +16,27 @@
 	 * color can be provided to set custom background color for active tab.
 	 * @type {string}
 	 */
-	export let color = '#268aed'; // or any 6 digit hex color code.
-	const colorArray = hexToRgbArray(color);
+	export let color = '#268aed';
+	color = color.replace(/\s+/g, '').toLowerCase(); // clean string
 
-	function hexToRgbArray(hexColor) {
-		const hexDigits = hexColor.replace(/\s+/g, '').replace('#', '');
+	const bgColor = isValidColorString(color) ? addOpacityToColor(color) : '#268aed1a';
+	const borderColor = isValidColorString(color) ? color : '#268aed';
 
-		// Check if the provided color contains valid hex characters.
-		if (!/^[0-9a-fA-F]{6}$/.test(hexDigits)) {
-			return null;
-		}
+	function isValidColorString(inputColor) {
+		const hexRegex = /^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{6})$/;
+		const rgbRegex = /^rgb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/;
```

Comment:
is this regex case sensitive?

I believe "RGB(255,0,0)" is a valid css colour, for example

Reply:
Yes this is case sensitive. But "RGB(255,0,0)" works since prior to calling this function, the input string is converted into lowercase.

```svelte
color = color.replace(/\s+/g, '').toLowerCase(); // clean string
```